vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -252,10 +252,10 @@ enum RewardsType {
 };
 
 enum ReportType {
-  GRANT = 0,
+  GRANT_UGP = 0,
```

Comment:
See above discussion on `ReportType`, we should also be using enum classes

Reply:
do not want to introduced change like enum class in this one as it's not a refactor

Reply:
enum class was a nit, but please see `ReportType` discussion above